contents/blog/b2b-saas-product-metrics.md

```diff
@@ -0,0 +1,210 @@
+---
+date: 2022-06-14
+title: The most useful B2B SaaS product metrics (and how to measure them)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/give-back-friday.png
+featuredImageType: full
+categories: ["Guides", "PostHog Academy", "Product analytics"]
+author: ["andy-vandervell"]
+---
+
+So, you're building a B2B SaaS product.
+
+Everyone knows you've got to measure stuff to succeed. This is not news. 
+
+What, why, and how? These are the questions that matter.
+
+In this guide we will:
+
+1. Reveal what makes a useful metric for a B2B product
+2. Explain some of the common metrics used by B2B SaaS companies
+3. Help you choose the right metrics for your product
+
+This guide is about _product_ metrics, rather than revenue or wider business metrics. Naturally, they're important too.
+
+Oh and, spoiler alert, we build an open-source analytics platform called PostHog. It's basically Mixpanel + Hotjar + Optimizely + LaunchDarkly in one tool you can host yourself. 
```

Comment:
I reckon this should come later as it's a smidge too sales-y - you haven't justified to me yet why I should use a tool like PostHog in the article

Reply:
Would be interested in hearing alternative views on this. I put it here mainly in the interests of transparency, and tried to keep the language informal so it didn't feel too jarring. 

Alternatively, I could put it in the 'Recommended metrics' section as a footnote to that.

Reply:
They aren't, but we do replace them right? Sessions recording and feature flags.

Reply:
Yeah, we do. Just think it makes it less sales-y if we stick to what's strictly relevant.